_pages/code-of-conduct.md

```diff
@@ -5,6 +5,86 @@ title: Code of Conduct
+
+## Expected Behavior
+
+All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform, whether online or in-person. In order to foster a positive and professional learning environment, we encourage the following kinds of behaviours in all Carpentries events and platforms:
+
+* use welcoming and inclusive language that is appropriate for a diverse audience
+* be respectful of different viewpoints and experiences
+* act in good faith and assume that others do as well: e.g. gracefully accept constructive criticism
+* give everyone a chance to talk and contribute to activities and discussions
+* humor is appreciated; however, sexual, racist, or exclusionary jokes and imagery are never appropriate
+* recognize that intentions do not equal impact; be ready to apologize or correct yourself when this occurs
+
+## Unacceptable Behavior
+
+Examples of unacceptable behavior include:
+
+* written or verbal comments which have the effect of excluding people
+* insults or put downs
+* sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
+* sustained disruption of talks, events or communications
+* continuing to initiate interaction (including photography or recording) with someone after being asked to stop
+* the display of sexual or violent images
+* publication of private communication without consent
+* unwelcome sexual attention
+* nonconsensual or unwelcome physical contact
+* causing someone to fear for their safety, such as through stalking, following, or intimidation
+* violent threats or language directed against another person
+* incitement to violence, suicide, or self-harm
+
+## Enforcement
+
+Community members and workshop participants who are asked to stop any harassing or discriminatory behavior are expected to comply immediately. This applies to all of our workshops and events, as well as interactions that take place online, across e-mail, Etherpad, social media or in any other medium.
+
+If someone engages in behavior that violates this code of conduct, they may be warned, asked to leave the event or premises, or reported to other organizations for further investigation and sanctions (including, but not limited to The Carpentries Code of Conduct Committee, the University of Florida, or law enforcement). 
+
+If you are being harassed, notice that someone else is being harassed, or have any concerns, please let us know through one or more of these reporting options:
+
+**(Do note that some of contacts may be UF employees who are mandatory reporters. If you have concerns about privacy or escalation of certain incidents, you may wish to use the anonymous form or check with the instructor or organizer of the event before reporting.)**
```

Comment:
I'd love to link "mandatory reporters" to a link to UF's mandatory reporting policy, but I couldn't find it. The closest I got is our [sexual harassment policy](https://hr.ufl.edu/forms-policies/policies-managers/sexual-harassment/), which says: "Any employee, student, or visitor who has knowledge of an occurrence of sexual misconduct or questions regarding the policy is strongly encouraged to promptly reach out to the following party [...] Failure to comply with this policy could result in disciplinary action, up to and including disaffiliation or termination as deemed appropriate by the adjudicating body."

Reply:
I wasn't able to dig out a description either, but it's definitely a component of the required training for, I'm guessing staff (incl. postdocs) and faculty.